src/modules/utils/timestampUtils/timestampUtils.ts

```diff
@@ -0,0 +1,24 @@
+export const formatDate = (unixTimestamp?: number): string => {
```

Comment:
Should this just take a unix printout for now then as placeholder?

Reply:
I would remove it altogether, allow the user to pass in a string (temporarily), and add a todo where the date is being used. See the `ProposalDataListItemStatus` component to see what I did in there. I understand it does require more work when the formatter is introduced since we have to make changes both in the ODS and the App, but the counter to that is the flexibility. Also, we need to figure out whether it makes sense to have the date formatter in the core vs the modules